packages/web-app-files/src/views/LocationPicker.vue

```diff
@@ -299,18 +299,12 @@ export default {
     currentHint() {
       if (this.currentAction === 'move') {
         return this.$gettext(
-          `Navigate into the desired folder and move selected resources into it.
-          You can navigate into a folder by clicking on its name.
-          To navigate back, you can click on the breadcrumbs.
-          Resources will be moved into the folder where you are currently located.`
+          'Navigate into the desired folder and move selected resources into it. You can navigate into a folder by clicking on its name. To navigate back, you can click on the breadcrumbs. Resources will be moved into the folder where you are currently located.'
         )
       }
 
       return this.$gettext(
-        `Navigate into the desired folder and copy selected resources into it.
-        You can navigate into a folder by clicking on its name.
-        To navigate back, you can click on the breadcrumbs.
-        Resources will be copied into the folder where you are currently located.`
+        'Navigate into the desired folder and copy selected resources into it. You can navigate into a folder by clicking on its name. To navigate back, you can click on the breadcrumbs. Resources will be copied into the folder where you are currently located.'
       )
     }
   },
```

Comment:
Just for the record: this is probably not invalid per se, just our lib and/or transifex just don't deal well with this.

Reply:
i wonder why, cant it handle \n?

Reply:
I *think* our lib/tool creates files that look like this

``` 
msgid ""
"You can navigate into a folder by clicking on its name."
"..."
```

which is valid, but apparently transifex does not like the empty `""` prefix string, if I got it all right in my discussion with @kulmann

Reply:
... which sounds really weird. Honestly I'm not sure what to believe anymore regarding transifex 🙈

Reply:
Ah, I remember why I changed this 😆 it was never actual multiline text so we can also have it as one line to avoid the empty msgid start.